src/directory/directory.mjs

```diff
@@ -250,6 +250,9 @@ export const directory = {
                 {
                   path: 'src/pages/[platform]/build-a-backend/storage/authorization/index.mdx'
                 },
+                {
+                  path: 'src/pages/[platform]/build-a-backend/storage/upload/index.mdx'
```

Comment:
Was this added from a PR? It seems like this is causing the build to fail because the file doesn't exist. Should it be a gen1 link or something?

Reply:
That directory entry has been removed